src/tests/mod.rs

```diff
@@ -86,10 +107,16 @@ fn check_result(path_to_codebase: &Path, path_to_expected: &Path) {
       let expected_file_path = path_to_expected.join(file_name);
       let expected_content = read_file(&expected_file_path).unwrap();
 
-      if !eq_without_whitespace(&cb_content, &expected_content) {
-        all_files_match = false;
-        error!("{}", &cb_content);
+      if (ignore_whitespace && eq_without_whitespace(&cb_content, &expected_content))
+        || (cb_content
+          .trim_end()
+          .eq(&expected_content.trim_end().to_string()))
```

Comment:
This is some weird formatting overall for the full conditional expression, but since code is now auto-formatted, I am fine with it if this is what the tool does 😅

Reply:
No action needed here, as far as I am concerned 🙂